src/main/java/ai/task/Task.java

```diff
@@ -0,0 +1,68 @@
+package ai.task;
+
+/**
+ * Class that has a status and description.
+ */
+public class Task {
+    private boolean isDone = false;
+    private String description;
+
+    public Task(String description) {
+        this.description = description;
+    }
+
+    /**
+     * Creates a new Task object.
+     * @param description String to be stored as description.
+     * @param isDone whether the task is done.
```

Comment:
Maybe there should be an empty line between the description and the parameters? Same applies for all other JavaDoc comments :)